.werft/build.ts

```diff
@@ -939,13 +939,13 @@ async function addDNSRecord(namespace: string, domain: string, isLoadbalancer: b
     werft.done(installerSlices.DNS_ADD_RECORD);
 }
 
-export async function issueMetaCerts(namespace: string, domain: string) {
+export async function issueMetaCerts(name: string, namespace:string, domain: string) {
```

Comment:
I would prefer naming the arguments `previewNamespace` and `certsNamespace` or something to that affect so it is clear that both args are namespace names. I also generally prefer option arguments so we get named parameters, e.g. 

```ts
export async function issueMetaCerts(options: {previewNamespace: string, certsNamespace: string, domain: string})
```

Reply:
Good suggestion! applied :)